src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        Colors[] colors = Colors.values();
+        int index = random.nextInt(colors.length);
+        Colors color = colors[index];
+        return color.name().toLowerCase();
```

Comment:
```suggestion
        return colors[index].name().toLowerCase();
```

Reply:
I did this in the next commit, but I don’t understand why it needs to be done this way